### PR TITLE
added module to return the bbox for the signature

### PR DESCRIPTION
**Pull request by Purefekt**
I added a module in src/ called bounding_boxes which uses the cropper module but instead of cropping the image, it returns the x,y,w,h which was detected and later cropped. i made this change personally when using this package. 
It is a good idea since for some cases we need the bounding boxes, specially for when testing the accuracy on a huge dataset where we might have the ground truth bboxes and we want to compare that to the calculated bboxes using intersection over union ratio. 
I have not made too many changes to the docstrings so that needs changing. I have made a small check where you can see its working fine.

### CODE REVIEW

**Comment by EnzoSeason** [comment]
Thank you for your contribution! Nice work on .tif files! 
You're right. The coordinates of the regions are useful. I'll check your codes!

File: check_bbox/test.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+from signature_detect.loader import Loader
+from signature_detect.cropper import Cropper
+from signature_detect.extractor import Extractor
+from signature_detect.bounding_box import Bounding_box
+import cv2
+
+path = '/Users/veersingh/Desktop/docs_with_signs_dataset(tobacco800)/images/aam09c00.tif'
```
Comment (EnzoSeason) [inline check_bbox/test.py:7]: The absolute path make the tests difficult on my side :(

File: src/signature_detect/bounding_box.py [hunk 2]
```diff
@@ -0,0 +1,189 @@
+import math
```
Comment (EnzoSeason) [inline src/signature_detect/bounding_box.py:1]: Avoiding changing the whole class. It's hard to see your modification.

File: src/signature_detect/bounding_box.py [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+            image = Image.fromarray(mask)
+            cropped_image = image.crop((x, y, x + w, y + h))
+            cropped_mask = np.array(cropped_image)
+
+            results[key] = {
+                "cropped_region": cropped_region,
+                "cropped_mask": cropped_mask,
+            }
+
+        x = results[0]['cropped_region'][0]
```
Comment (EnzoSeason) [inline src/signature_detect/bounding_box.py:170]: Ahhh! I get your point! Maybe we can create two public functions, one for returning cropped region, and the other returns cropped mask

File: check_bbox/test.py (hunk 1 above)
Comment (Purefekt) [reply check_bbox/test.py:7]: That is my bad. 
`path = 'aam09c00.tif'`
should work

File: src/signature_detect/bounding_box.py (hunk 2 above)
Comment (Purefekt) [reply src/signature_detect/bounding_box.py:1]: that is a sloppy contribution, so sorry. I basically copied pasted from my code :/

File: src/signature_detect/bounding_box.py (hunk 3 above)
Comment (Purefekt) [reply src/signature_detect/bounding_box.py:170]: Yes!!! :)

**Comment by Purefekt** [comment]
I made a mistake. I commited with my work account which was set on global. Please dont merge this PR

**Comment by EnzoSeason** [comment]
okay! But still, it's welcome to make a contribution!

**Comment by Purefekt** [comment]
i will redo with my personal account

**Pull request closed without merging**